Formula/kde-ki18n.rb

```diff
@@ -1,8 +1,9 @@
 class KdeKi18n < Formula
   desc "KDE Gettext-based UI text internationalization"
   homepage "https://api.kde.org/frameworks/ki18n/html/index.html"
-  url "https://download.kde.org/stable/frameworks/5.76/ki18n-5.76.0.tar.xz"
-  sha256 "0e87bc1136e21f7860f15daa39e8d16e5a773995fce2b87b0cef0043c4ce0e7a"
+  url "https://download.kde.org/stable/frameworks/5.77/ki18n-5.77.0.tar.xz"
+  sha256 "b2e1b74dedc1a3af88f04c470922d1fafb892d5846ea91ad139d421070cff357"
+  license all_of: ["BSD-3-Clause", "LGPL-2.0-or-later", "LGPL-2.1-only", "LGPL-3.0-only"]
```

Comment:
[`src/klocalizedtranslator.h`](https://invent.kde.org/frameworks/ki18n/-/blob/master/src/klocalizedtranslator.h), [`src/klocalizedtranslator.cpp`](https://invent.kde.org/frameworks/ki18n/-/blob/master/src/klocalizedtranslator.cpp), [`autotests/ktranscripttest.cpp`](https://invent.kde.org/frameworks/ki18n/-/blob/master/autotests/ktranscripttest.cpp), [`autotests/ktranscripttest.h`](https://invent.kde.org/frameworks/ki18n/-/blob/master/autotests/ktranscripttest.h), [`autotests/ktranscriptcleantest.h`](https://invent.kde.org/frameworks/ki18n/-/blob/master/autotests/ktranscriptcleantest.h), and [`autotests/ktranscriptcleantest.cpp`](https://invent.kde.org/frameworks/ki18n/-/blob/master/autotests/ktranscriptcleantest.cpp) all have the following line:

```
SPDX-License-Identifier: LGPL-2.1-only OR LGPL-3.0-only OR LicenseRef-KDE-Accepted-LGPL
```

Same thoughts about `LicenseRef-KDE-Accepted-LGPL` as in `kde-kdoctools`

`LGPL-2.0-only` and `LGPL-3.0-only` only appear in these files. Therefore, I think the license should be:

```ruby
  license all_of: [
    "BSD-3-Clause",
    "LGPL-2.0-or-later",
    any_of: ["LGPL-2.1-only", "LGPL-3.0-only"],
  ]
```